assets/src/scripts/monitoring-location/components/network-list.js

```diff
@@ -20,21 +20,19 @@ const buildNetworkURL = function(link) {
 
 const addNetworkRows = function(node, {hasData, networkList}){
     if (hasData){
-        const input = node.select('#network-list-table');
-        input.append('thead').html(
-            '<tr><th>Name</th><th>Link</th></tr>'
-        );
-        const tbody = input.append('tbody');
+        const input = node.append('ul')
+        .classed('usa-fieldset', true)
+        .classed('usa-list--unstyled', true);
 
-        let tbodyHTML = '';
+        let ulHTML = '';
         let networkUrl;
         networkList.forEach(function(network) {
             networkUrl = buildNetworkURL(network.href);
-            tbodyHTML += `<tr><td>${network.title}</td><td><a href="${networkUrl}">
-                ${networkUrl}</a></td></tr>`;
+            ulHTML += `<li><a class="usa-link" href="${networkUrl}">
```

Comment:
I would prefer that d3.append and other d3 functions be used to render the html rather than building the tags. This is the pattern we follow in other places.

Reply:
I changed this to be d3 selection oriented.